templates/includes/js.jade

```diff
@@ -1,5 +1,14 @@
 //- js lazysizes
 script(src=contents.js.vendor['lazysizes.min.js'].url)
+//- plyr 
+script(Src=contents.js['plyr.js'].url)
```

Comment:
@nathakits plyr only needs to be on the `/pupil` page. This would load plyr on every page. Please make a block in the `/pupil` page template and only load this script for that page/route.

Reply:
See the `page_scripts` block in the `vr-ar.jade` template for reference on how to do this.